### ISSUE TITLE
Unable to focus the table toolbar when an image is inserted into a table cell

### ISSUE THREAD

**Issue by mabryl** [ckeditor/ckeditor5#14204]
## 📝 Provide detailed reproduction steps (if any)

1. Create a table that consist of a single cell
2. Insert an image into the cell
3. Try to access the table toolbar

### ✔️ Expected result

You should be able to focus the table toolbar and change the options available there.

### ❌ Actual result

You can only focus the image toolbar:

![image](https://github.com/ckeditor/ckeditor5/assets/72079603/21ff952d-c202-47e7-8aae-fa140c1fba14)

You can switch the focus to the table toolbar with the `Alt + F10` (for Windows) or `Option + F10` (for macOS) keyboard shortcut but not by using the mouse.

## 📃 Other details

* Browser: cross-browser
* OS: cross-OS
* First affected CKEditor version: N/A
* Installed CKEditor plugins: N/A

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.
